modules/run/artifacts/opt/jboss/container/java/run/run-java.sh

```diff
@@ -3,6 +3,13 @@
 # Fail on a single failed command
 set -eo pipefail
 
+#These are defined explicitly here to avoid defining them in templates/jlink/Dockerfile
+export JBOSS_CONTAINER_UTIL_LOGGING_MODULE="/opt/jboss/container/util/logging"
```

Comment:
I think we should either use substring expansion to only set the value if its not already set, e.g.
`export JBOSS_CONTAINER_UTIL_LOGGING_MODULE="${JBOSS_CONTAINER_UTIL_LOGGING_MODULE-/opt/jboss/container/util/logging}"`
or just replace the variable with the value everywhere.

There are some old unit tests using [bats](https://github.com/bats-core/bats-core) in the source tree which rely up setting this variable to a different value. Those tests are currently broken for other reasons. We might remove them, or fix them. But until we decide I guess we should not hardcode the value, so please use substring expansion so the value can be overridden.